From a266ac62ea1e4deb7edd9332b87b29097607d7c7 Mon Sep 17 00:00:00 2001
From: Allison Ryan Lortie <desrt@desrt.ca>
Date: Thu, 26 Nov 2015 09:25:39 -0500
Subject: [PATCH] tests: fix a test on 32-bit builds

We were trying to squeeze 64-bit test vectors into gsize, which is fine
on 64bit systems but doesn't work very well on 32-bit.

Change that to a guint64.
Applied-Upstream: 2.47.4
---
 glib/tests/overflow.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/glib/tests/overflow.c b/glib/tests/overflow.c
index f241abd..5c68d95 100644
--- a/glib/tests/overflow.c
+++ b/glib/tests/overflow.c
@@ -16,7 +16,7 @@
 typedef struct
 {
   gboolean success;
-  gsize c, a, b;
+  guint64 c, a, b;
 } Case;
 
 static void
-- 
2.6.2

